functions/hash_to_opts.pp

```diff
@@ -25,18 +27,25 @@ function simplib::hash_to_opts (
   Struct[{
     Optional[connector]       => String[1],
     Optional[prefix]          => String[1],
+    Optional[array_style]     => Enum['comma','repeat'],
```

Comment:
I would change this to `delimiter` and `repeat`.

Reply:
Those are better names, I like them